boards/silabs/radio_boards/siwx917_rb4338a/siwx917_rb4338a.dts

```diff
@@ -20,6 +20,36 @@
 		zephyr,console = &ulpuart;
 		zephyr,shell-uart = &ulpuart;
 	};
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		sw0 = &button0;
+		sw1 = &button1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
```

Comment:
newline

Reply:
done. I have also checked the other `.dts` introduced in this PR. I believe everything is fine now.